doc/source/migration_details.rst

```diff
@@ -5,36 +5,126 @@ Check out the code manually and upgrade your database by calling odoo-bin,
-   available in the addons path in the Odoo instance of the new version.
-
-   Or, for older versions: check out the OpenUpgrade source code from Github
-   for the branches you need. Each branch migrates to its version from the
-   previous version, so branch 13.0 migrates from 12.0 to 13.0. If you are
-   migrating across multiple versions, you need to run each version of
-   OpenUpgrade in order. Skipping versions is not supported.
-
-2. Check if there are migration scripts provided for the set of modules that
-   are installed in your Odoo database. If there are modules for which no
-   migration scripts have been developed yet, your migration may fail or the
-   integrity of your database may me lacking. Check the module coverage in
-   this documentation.
-
-3. Decide which database you are going to upgrade. You absolutely *must* make a
-   backup of your live database before you start this process!
-
-4. Edit the configuration files and command line parameters to point to the
-   database you are going to upgrade. Run Odoo with the *--update all
-   --stop-after-init --load=web,openupgrade_framework* flags.
-
-   For versions earlier than 14.0 that are running the OpenUpgrade fork rather
-   than Odoo itself, you do not pass the `load` parameter.
-
-5. Run the upgrade and check for errors. You will probably learn a lot about
-   your data and have to do some manual clean up before and after the upgrade.
-   Expect to repeat the process several times as you encounter errors, clean up
-   your data, and try again. If necessary, ask for help or report bugs on
-   Github.
+1. Get the code from OpenUpgrade and dependencies
+*************************************************
+
+Make the `openupgrade_framework` and the `openupgrade_scripts` modules
+available in the addons path in the Odoo instance of the new version.
+
+Or, for older versions: check out the OpenUpgrade source code from Github
+for the branches you need. Each branch migrates to its version from the
+previous version, so branch 13.0 migrates from 12.0 to 13.0. If you are
+migrating across multiple versions, you need to run each version of
+OpenUpgrade in order. Skipping versions is not supported.
+
+2. Check coverage of the migration scripts for your installed modules
+*********************************************************************
+
+Check if there are migration scripts provided for the set of modules that
+are installed in your Odoo database. If there are modules for which no
+migration scripts have been developed yet, your migration may fail or the
+integrity of your database may me lacking. Check the module coverage in
+this documentation.
+
+At this stage, for some of the uncovered modules, you might want to uninstall
+them from your database and re-install it later on after the migration.
+This option might reduce the workload of the migration and can be applicable
+for "ux" modules like ``web_widget_image_download`` if you did not
+override this modules.
```

Comment:
This advice seems dangerous. Uninstalling modules from Odoo can cause data loss.

Reply:
Could we place this in a section / page "Dangerous optional action with possible benefits"?

Inside that section we could also add the part relate to ``fsync = off``

Or would you prefer to remove it completely?

Reply:
Maybe a new page under "https://doc.therp.nl/openupgrade/development.html" ?

Reply:
For me the fsync thing should never be mentioned. If you need it, it's because OpenUpgrade is not well optimized, to add we have to focus on that instead.

Reply:
@jcdrubay yes, for development purposes I myself run a partial migration and then execute
`update ir_module_module set state = 'to remove' where state IN ('to upgrade', 'to install') returning name`
I then run Odoo with `-u dummy --stop-after-init` to uninstall the modules that could not be upgraded yet. That way, you get a partially migrated database that you can login to and perform UI testing. So mentioning this under the `development` section sounds good as long as it is clear that this step does involve data loss.